myriad-scheduler/src/main/java/org/apache/myriad/scheduler/ServiceTaskFactoryImpl.java

```diff
@@ -126,13 +112,13 @@ public TaskInfo createTask(Offer offer, FrameworkID frameworkId, TaskID taskId,
 
     LOGGER.info("Command line for service: {} is: {}", nodeTask.getTaskPrefix(), strB.toString());
 
-    Scalar taskMemory = Scalar.newBuilder().setValue(nodeTask.getProfile().getMemory()).build();
-    Scalar taskCpus = Scalar.newBuilder().setValue(nodeTask.getProfile().getCpus()).build();
-
     TaskInfo.Builder taskBuilder = TaskInfo.newBuilder();
 
-    taskBuilder.setName(nodeTask.getTaskPrefix()).setTaskId(taskId).setSlaveId(offer.getSlaveId()).addResources(Resource.newBuilder().setName("cpus").setType(Value.Type.SCALAR).setScalar(taskCpus).build()).addResources(Resource.newBuilder()
-        .setName("mem").setType(Value.Type.SCALAR).setScalar(taskMemory).build());
+    taskBuilder.setName(nodeTask.getTaskPrefix())
```

Comment:
Lines 120 and 121 were the only lines here that aren't formatting, I could remove the formatting.